frontend/src/locales/en.yml

```diff
@@ -426,6 +426,16 @@ issue:
   stage-select:
     active: "{name} (active)"
   not-allowed-to-single-database-in-tenant-mode: Can't {operation} to single database in tenant mode
+  migration-mode:
+    title: Migration mode
+    normal:
+      title: Normal migration
+      description: Perform schema change directly on target table.
+    online:
+      title: Online migration
+      description: >-
+        Perform schema change on a duplicated table. Switch table names after
+        data synchronization. {link}
```

Comment:
```suggestion
        Perform schema change on a duplicated empty table. Sync the data and swap the table name after sync finishes. Applying migration on a large table directly would cause long-time unavailability {link}.
```